test-sbt/native/src/main/scala/zio/test/sbt/ZTestRunnerNative.scala

```diff
@@ -138,7 +138,6 @@ sealed class ZTestTask(
         case _                   =>
       }
     }(Unsafe.unsafe)
-    scala.scalanative.loop.EventLoop.run()
```

Comment:
This was the fix to the issue that tests were not being executed and test results were not being reported. You need to change the code so that it now works without the event loop